### PR TITLE
svtav1_dilated_ren

### PR DESCRIPTION
**Pull request by kaileh57**
# submission name:

svtav1_dilated_ren

# upload zipped `archive.zip`

https://github.com/kaileh57/comma_video_compression_challenge/releases/download/archive/archive.zip

# report.txt

=== Evaluation results over 600 samples ===
  Average PoseNet Distortion: 0.08210164
  Average SegNet Distortion: 0.00475505
  Submission file size: 804,054 bytes
  Original uncompressed size: 37,545,489 bytes
  Compression Rate: 0.02141546
  Final score: 100*segnet_dist + √(10*posenet_dist) + 25*rate = 1.92

# does your submission require gpu for evaluation (inflation)?

yes (runs on CPU too, just slower)

# did you include the compression script? and want it to be merged?

yes

# additional comments

SVT-AV1 v2.3.0, preset 0, CRF 36, film-grain=22, keyint=180, scd=0, encoded at 544x408 (~47% Lanczos downscale).

Decoder bundles a 20,259-parameter Residual Enhancement Network (REN). The key architectural choice: dilated convs at dilation 1/2/4 instead of plain 3x3s, giving ~3x the effective receptive field at the same parameter count. Compression artifacts at CRF 36 that hurt SegNet and PoseNet tend to be structural (edges, silhouettes, lane markings) rather than pixel-local, so the wider receptive field matters. This was the single largest win in ablations: 1.92 vs 2.03 for a plain 4-conv REN with identical parameter count and training recipe.

REN runs at 512x384 (the task models' internal input resolution): bicubic upscale from encoded size to 512x384, REN forward pass, bicubic upscale to 1164x874.

Training: 25 epochs on 999 frame pairs (200-pair held-out val, val-proxy checkpoint saving), cosine LR from 5e-5, gradient clipping 1.0. Loss is normalized SegNet KL + PoseNet MSE + 0.01 temporal L1, with weights set to 1/baseline_distortion so each term starts at ~1.0 and needs no hand-tuning. REN weights are int8-quantized per-tensor with float32 scales and bz2-compressed to ~21 KB, included in archive.zip alongside the mkv.

### CODE REVIEW

**Comment by YassineYousfi** [comment]
Can you add the training code? I think it's more useful to merge with that than without.

**Comment by kaileh57** [comment]
> Can you add the training code? I think it's more useful to merge with that than without.

Added an amalgamation of my training notebooks! Trained on Collab on a T4.

**Comment by YassineYousfi** [comment]
@kaileh57 Congratulations on making it onto the leaderboard! I hope you had fun competing in the challenge and learned something new along the way. If you are looking for a job or internship, please email givemeajob@comma.ai with a link to this PR. See you in the next challenge!
